hazelcast/include/hazelcast/client/ClientConfig.h

```diff
@@ -395,10 +405,10 @@ namespace hazelcast {
 
             std::map<std::string, config::ReliableTopicConfig> reliableTopicConfigMap;
 
-            std::map<std::string, config::NearCacheConfig> nearCacheConfigMap;
+            util::SynchronizedMap<std::string, config::NearCacheConfig> nearCacheConfigMap;
```

Comment:
Why do we need synchronisation here ?. Isn't this populated at the start only ? After that, it should read only.